app/views/content/three-things-to-help-you-get-into-teaching.md

```diff
@@ -8,7 +8,7 @@
   layout: layouts/campaigns/landing_page_with_hero_nav
   hero_nav:
     Receive personalised email updates and tips: "#receive-personalised-email-updates-and-tips"
-    Speak to current teachers at a teaching event: "#speak-to-current-teachers-at-a-teaching-event"
+    Speak to current teachers at a teacher training event: "#speak-to-current-teachers-at-a-teaching-event"
```

Comment:
Will this jump link need updating as the h2 it is jumping to is changing?

(It works fine in the Review app though?)

Reply:
@peteryates can you confirm whether this jump link needs changing or not?

Reply:
In this instance we don't need to because the `id` (bit after the colon) is hardcoded.